brave/browser-laptop
# Adds non-verified option to the payment page

Resolves #8231

Auditors: @mrose17 @bsclifton

### Test Plan
- go to payments page and enable payments
- go to some verified and non-verified sites (brianbondy.com is verified)
- go to payments page
- click on advance settings
- disabled option "Allow contributions to non-verified sites"
- all non-verified sites should be hidden from a table
